ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -56,10 +57,8 @@ Verify Pod Toleration
 Verify RHODS User Groups
     [Documentation]    Verify User Configuration after the upgrade
     [Tags]      Upgrade
-    ${admin}        Set Variable        ${payload[0]['spec']['groupsConfig']['adminGroups']}
-    ${user}     Set Variable        ${payload[0]['spec']['groupsConfig']['allowedGroups']}
-    Should Be Equal As Strings      '${admin}'      'rhods-admins,rhods-users'
-    Should Be Equal As Strings      '${user}'       'system:authenticated'
+    AdminGroups In OdhDashboardConfig CRD Should Be     rhods-admins
```

Comment:
This seems like a functional change - the `rhods-users` missing - is this truly what the test should expect? :thinking: 
disclaimer: I didn't really check the rest of the test.

Reply:
https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot#L78

in the pre-upgrade "rhods-users" is set as Admin group, so the post-upgrade one should consider it. i.e., @jstourac might be right

Reply:
I reverted test file `0201__pre_upgrade.robot` so @asanzgom can fix it in his PR 2222.

I kept only the fix for `ODHDashboardSettings.resource` as the logic there was wrong.